README.md

```diff
@@ -2,101 +2,22 @@
 
 ![labbox-ephys-install-tests](https://github.com/laboratorybox/labbox-ephys/workflows/labbox-ephys-install-tests/badge.svg?branch=master)
 
-**Under development** (and seeking helpers for this open source project)
+**Under development**
 
 Analysis and visualization of neurophysiology recordings and spike sorting results.
 
-Experimental jupyterlab extension: [labbox_ephys_widgets_jp](doc/labbox_ephys_widgets_jp.md)
+You can either run labbox-ephys as a JupyterLab extension or as a web server
 
-## Installation and basic usage
+JupyterLab extension: [labbox_ephys_widgets_jp](doc/labbox_ephys_widgets_jp.md)
```

Comment:
Reorganize the install docs to put web gui and jupyter extension on equal footing